packages/evm/src/evm.ts

```diff
@@ -296,8 +304,8 @@ export class EVM implements EVMInterface {
     }
 
     // Safeguard if "process" is not available (browser)
-    if (typeof process?.env.DEBUG !== 'undefined') {
-      this.DEBUG = true
+    if (process === undefined || opts.printCLDebugOutput === false) {
```

Comment:
I think we do not want this to "activate" DEBUG by this `printCLDebugOutput`, since this has no value on it's own and is only useful if `DEBUG` is used on the command line. So this leads to cases where debugging "activates" if the option is set to true and `process` is available, this is not adding any value but rather in doubt consumes performance.

The `printCLDebugOutput` option should really just be a - completely separated - guard towards the `DEBUG` usage on the command line.

Please switch the conditional logic back to the "old"/proposed way here, so leaving `  readonly DEBUG: boolean = false` and then set `true` as the default value for `printCLDebugOutput`.

Thanks! 🙂 🙏